src/plugins/gui/scenes/RewardsCardDashboardScene.tsx

```diff
@@ -97,6 +97,7 @@ export interface RewardsCardProps {
 export const RewardsCard = (props: RewardsCardProps) => {
   const { item, onPress, onRemovePress, shouldStack = false } = props
   const theme = useTheme()
+  const purchaseAmount = `$${item.amount.toString()}`
```

Comment:
The currency symbol should be part of the item object set by the provider. Useful for when we have nonUSD cards

Reply:
There is no API for this from Ionia's side, so there is no clear model yet. Since we're launching in the U.S. and with only USD, then why are we worry about a this in our model today?

I say we keep this hard-coded and include this change in the scope of another task when we open up to other countries with other fiat currencies supported.